Added accessibleLinks() method in theme-a11y (#62)
Shopify/theme-scripts: packages/theme-a11y/README.md

- [theme-a11y.min.js](http://unpkg.com/@shopify/theme-a11y@latest/dist/theme-a11y.min.js)

These files make Theme A11y accessible via the `Shopify.theme.a11y` global variable.

---

### accessibleLinks(messages, targetLinks, prefix)

> Add a preventive message to external links and links that open to a new window.

Reviewer: ```suggestion
Add a descriptive message to external links and links that open to a new window.
```

> This is being used to provide better accessibility experience.

Reviewer: ```suggestion

```

Sentence not needed - all of these helpers are here to provide a better experience 😄

- `messages`: Custom messages object to overwrite with keys: newWindow, external, newWindowExternal
- `messages.newWindow` - When the link opens in a new window (e.g. `target="_blank"`)

> - `messages.external` - When the link is external (e.g. www.cnn.com)

Reviewer: ```suggestion
- `messages.external` - When the link is to a different host domain.
```

I suggest taking out the specific domain examples. What if CNN wants to use us one day? 😛

> - `messages.newWindowExternal` - When the link is external and opens in a new window (`<a href="http://www.shopify.com" target="_blank">Shopify</a>`)

Reviewer: ```suggestion
- `messages.newWindowExternal` - When the link is to a different host domain and opens in a new window.
```
